src/content/docs/synthetics/synthetic-monitoring/private-locations/containerized-private-minion-cpm-configuration.mdx

```diff
@@ -12,6 +12,10 @@ redirects:
 freshnessValidatedDate: never
 ---
 
+<Callout variant="important">
+  On October 22, 2024, we will [end of life](/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm) the containerized private minion (CPM) and the legacy synthetics runtime versions it supports. Please review our [recommended migration steps](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide/#monitorMigration) to avoid degradation of your private location monitors. 
```

Comment:
"...and the legacy synthetics runtime versions it supports." Does "it" refer to CPM or the runtimes? AFAICT the runtime supports CPM, and we're wanting to EOL the runtime which means we also need to EOL CPM because of that. Is this accurate? (affects how I reword this)